tests/src/main/java/net/neoforged/neoforge/debug/entity/EntityEventTests.java

```diff
@@ -69,4 +71,18 @@ static void entityAttributeModificationEvent(final DynamicTest test, final Regis
                         donkey, d -> d.getAttribute(testAttr.get()).getValue(), "test attribute", 1.5D))
                 .thenSucceed());
     }
+
+    // @todo not sure if needed to turn this into a game test
```

Comment:
If your PR is not finished and is in progress, like what this TODO says, mark your PR as Draft. There is this button here that says Convert to Draft. Use that and take your PR out of draft when fully finished and ready for approvals
![image](https://github.com/neoforged/NeoForge/assets/40846040/7f18456a-c011-4485-b222-66b793d1c6e7)